doc/recipe/overview.rst

```diff
@@ -113,22 +113,25 @@ case the two experiments have different ensemble names
 ESMValTool also supports a simplified syntax to add multiple ensemble members from the same dataset.
 In the ensemble key, any element in the form `(x:y)` will be replaced with all numbers from x to y (both inclusive),
 adding a dataset entry for each replacement. For example, to add ensemble members r1i1p1 to r10i1p1
-you can use the following abreviatted syntax:
+you can use the following abbreviated syntax:
 
 .. code-block:: yaml
 
     datasets:
-      - {dataset: CanESM2, project: CMIP5, exp: historical, ensemble: r(1:10)i1p1, start_year: 2001, end_year: 2004}
+      - {dataset: CanESM2, project: CMIP5, exp: historical, ensemble: "r(1:10)i1p1", start_year: 2001, end_year: 2004}
 
 It can be included multiple times in one definition. For example, to generate the datasets definitions
 for the ensemble members r1i1p1 to r5i1p1 and from r1i2p1 to r5i1p1 you can use:
 
 .. code-block:: yaml
 
     datasets:
-      - {dataset: CanESM2, project: CMIP5, exp: historical, ensemble: r(1:5)i(1:2)p1, start_year: 2001, end_year: 2004}
+      - {dataset: CanESM2, project: CMIP5, exp: historical, ensemble: "r(1:5)i(1:2)p1", start_year: 2001, end_year: 2004}
```

Comment:
This doesn't seem to work for me, have you tried this?

Reply:
@bouweandela thanks. It seems that neither r(1:5)i(1:2)p1 nor  “r(1:5)i(1:2)p1” works. I am wondering what is the use case for this syntax because it had been introduced here.

Reply:
Ok, I pushed some changes to this branch to make it work. I guess @jvegasbsc intended to implement this, but forgot to do it.